htdocs/compta/paiement/cheque/card.php

```diff
@@ -544,7 +544,7 @@
 		if (count($lines[$bid])) {
 			foreach ($lines[$bid] as $lid => $value) {
 				print '<tr class="oddeven">';
-				print '<td>'.dol_print_date($value["date"], 'day').'</td>';
+				print '<td>'.dol_print_date($value["paymentdate"], 'day').'</td>';
```

Comment:
It seems the label depends on $type
```
if ($type == 'CHQ') {
			print $langs->trans("DateChequeReceived");
		} else {
			print $langs->trans("DatePaymentForDepositReceived", $type);
		}
```
So may be the value should also depend on $type ?

Reply:
Hello @eldy 
The label depends on $type it's true. Can be CHQ or FAC or GIR etc...

But the problem stays the same, whatever the type is. 
At the moment, the list displays $value["date"] => $obj->datec . We need to see the payment date. 

I've run some tests with $type == 'FAC' and 'LIQ' with $value["paymentdate"] it's working, no needs to do a IF statement. The correct value is written in the database always at the same spot llx_bank -> dateo .

By the way DatePaymentForDepositReceived don't have translation at all.